types/mongoose-deep-populate/mongoose-deep-populate-tests.ts

```diff
@@ -2,15 +2,16 @@ import mongooseDeepPopulate from 'mongoose-deep-populate';
 import * as mongoose from 'mongoose';
 import { Schema } from 'mongoose';
 
-var connection = mongoose.connect("mongodb://localhost/myDatabase");
+async function wrapper() {
+    var connection = await mongoose.connect("mongodb://localhost/myDatabase");
+    var deepPopulate = mongooseDeepPopulate(connection);
 
-var deepPopulate = mongooseDeepPopulate(connection);
+    var bookSchema = new Schema({
```

Comment:
Excuse me, it's probably a superfluous catch but, can _vars_ be replaced by _consts_?